source/grid/grid_in.cc

```diff
@@ -757,6 +757,31 @@ void GridIn<dim, spacedim>::read_ucd (std::istream &in)
   tria->create_triangulation_compatibility (vertices, cells, subcelldata);
 }
 
+namespace
+{
+  // Forward declaration of helper class
+  template <int dim>
+  class Abaqus_to_UCD;
```

Comment:
I can do but then one be dumping quite a big, awkward class in among the other `GridIn` functions. What about putting the complete class declaration here and the definition for its functions at the bottom of `grid_in.cc` similar to what I already have done?